libs/damap/src/lib/shared/search-field/search-field.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-form-field [appearance]="appearance" class="search-field">
+  <!-- translate has to happen here, if it is done outside label is rendered as empty string always-->
+  <mat-label *ngIf="label">{{ label | translate }}</mat-label>
+  <input
```

Comment:
It would be great if, when the mat-form-field is in outline active mode, the search icon also turns 'active' with the primary color for better visual alignment.
![icon -active](https://github.com/user-attachments/assets/b239bc3e-3f6d-4e1f-b994-df7ffe326e5c)

Reply:
Will do, together with changing the border color when not focused.